test/type.js

```diff
@@ -0,0 +1,160 @@
+'use strict';
+
+import assert from 'assert';
+import sinon from 'sinon';
+import typeValidators from '../src/type';
+
+describe('Type', function() {
+	beforeEach(function() {
+		console.error = sinon.stub();
```

Comment:
Oh, and just to explain, this wouldn't make any difference when running this test file alone, but it will when running it together with others (in the ci for example).